hack/update-build.sh

```diff
@@ -0,0 +1,63 @@
+
+#!/usr/bin/env bash
+
+echo "Updating container file"
+
+: "${COMMIT:=$(git rev-list --abbrev-commit --tags --max-count=1)}"
+: "${CONTAINER_FILE:=./Dockerfile}"
+: "${BUNDLE_CONTAINER_FILE:=./bundle.Dockerfile}"
+: "${CATALOG_CONTAINER_FILE:=./catalog.Dockerfile}"
+
+
+supported_ocp_versions="v4.13"
+manifests_dir="./bundle/manifests"
+metadata_dir="./bundle/metadata"
+crd_name="flows.netobserv.io_flowcollectors.yaml"
+crd_file="${manifests_dir}/${crd_name}"
+
+
+cat <<EOF >>"${CONTAINER_FILE}"
+LABEL com.redhat.component="network-observability-operator-container"
+LABEL name="network-observability-operator"
+LABEL io.k8s.display-name="Network Observability Operator"
+LABEL io.k8s.description="Network Observability Operator"
+LABEL summary="Network Observability Operator"
+LABEL maintainer="support@redhat.com"
+LABEL io.openshift.tags="network-observability-operator"
+LABEL upstream-vcs-ref="${COMMIT}"
+LABEL upstream-vcs-type="git"
+EOF
+
+echo "Updating fbc bundle file"
+
+
+cat <<EOF >>"${BUNDLE_CONTAINER_FILE}"
+LABEL com.redhat.component="network-observability-operator-bundle-container"
+LABEL name="network-observability-operator-bundle"
+LABEL io.k8s.display-name="Network Observability Operator Bundle"
+LABEL io.k8s.description="Network Observability Operator Bundle"
```

Comment:
nit, for consistency with current description

```suggestion
LABEL io.k8s.display-name="Network Observability operator bundle"
LABEL io.k8s.description="Bundle for the Network Observability operator"
```